src/test/java/io/stargate/sgv2/jsonapi/api/v1/CountIntegrationTest.java

```diff
@@ -708,4 +712,13 @@ public void byBooleanColumn() {
           .body("errors", is(nullValue()));
     }
   }
+
+  @Nested
+  @Order(2)
+  class Metrics {
+    @Test
+    public void checkMetrics() {
+      CountIntegrationTest.super.checkMetrics("CountDocumentsCommands");
```

Comment:
This is fine, but we need unit tests that test the error cases, tenant exist yes/no, etc..

Reply:
Added MeteredCommandProcessorTest.java for unit testing